ssh_check/datadog_checks/ssh_check/ssh_check.py

```diff
@@ -22,6 +24,15 @@
 )
 
 
+def patch_paramiko():
+    # Patch to allow paramiko to use SHA-256 vs MD5
```

Comment:
add link here to PR where this is suggested. This is more robust than git blame.